src/sbl_filing_api/services/validators/filing_validators.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+from sbl_filing_api.entities.models.dao import FilingDAO
+from .base_validator import ActionValidator
+
+log = logging.getLogger(__name__)
+
+
+class CheckFilingExists(ActionValidator):
+    def __init__(self):
+        super().__init__("check_filing_exists")
+
+    def __call__(self, filing: FilingDAO, period_code: str, lei: str, **kwargs):
+        if filing:
+            return f"Filing already exists for Filing Period {period_code} and LEI {lei}"
+
+
+class CheckFilingNotExists(ActionValidator):
+    def __init__(self):
+        super().__init__("check_filing_not_exists")
+
+    def __call__(self, filing: FilingDAO, lei: str, period_code: str, **kwargs):
+        if not filing:
+            return f"There is no Filing for LEI {lei} in period {period_code}, unable to sign a non-existent Filing."
```

Comment:
These 2's name and logic are flipped compared to the voluntary filer, and contact info checks. My initial thought of these function names are passing conditions instead of failing conditions. We probably should rename all the checks to be more explicit then, like `FailIfFilingExists` and `FailIfFilingNotExists`; or maybe `InvalidIfFilingExists`, etc. if we want to make them pass conditions: `ValidIfFilingExists`, etc.; and we can make the other validators names explicit as well, i.e. `ValidVoluntaryFilerStatus`, or `InvalidVoluntaryFilerStatus`, etc.